Neos.Flow/Classes/Testing/RequestHandler/RuntimeSequenceHttpRequestHandler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Testing\RequestHandler;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Http\HttpRequestHandlerInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Everything from `RuntimeSequenceInvokingRequestHandler` applies to this.
```

Comment:
If you write it like this
```suggestion
 * Everything from {@see RuntimeSequenceInvokingRequestHandler} applies to this.
```
the class is linked and the doc comment will be adjusted automatically if it is renamed